src/main/java/com/microsoft/spring/data/gremlin/conversion/script/GremlinScriptLiteralGraph.java

```diff
@@ -60,19 +58,20 @@
     }
 
     @Override
+    @SuppressWarnings("unchecked")
     public List<String> generateUpdateScript(@NonNull GremlinSource source) {
         if (!(source instanceof GremlinSourceGraph)) {
             throw new GremlinUnexpectedSourceTypeException("should be the instance of GremlinSourceGraph");
         }
 
         final List<String> scriptList = new ArrayList<>();
         final GremlinSourceGraph sourceGraph = (GremlinSourceGraph) source;
-        final GremlinScriptLiteralVertex scriptVertex = new GremlinScriptLiteralVertex();
-        final GremlinScriptLiteralEdge scriptEdge = new GremlinScriptLiteralEdge();
+        final List<GremlinSource> vertexes = (List<GremlinSource>) sourceGraph.getVertexSet();
+        final List<GremlinSource> edges = (List<GremlinSource>) sourceGraph.getEdgeSet();
```

Comment:
Same as above.

Reply:
Seems generateUpdateScript and generateInsertScript are almost the same? Can these two methods be combined to handle 2 different(or more) cases?